config-sample.yaml

```diff
@@ -311,3 +309,11 @@ keys:
 #   # value less or equal than 0 means no limit.
 #   subscription_limit_video: 0
 #   subscription_limit_audio: 0
+#   # limit size of room and participant's metadata, 0 for no limit
+#   max_metadata_size: 0
+#   # limit size of participant attributes, 0 for no limit
+#   max_attributes_size: 0
```

Comment:
do we only care about total size of all attributes?

It might be more predictable for users if there were separate limits on 
1. nr of attributes
2. individual size of each attribute

Reply:
yeah, what's more important for operators are the size per participant. I thought about having both limits in place, but then it seemed more confusing.